config/application.rb

```diff
@@ -47,5 +47,13 @@ class Application < Rails::Application
     config.action_dispatch.rack_cache = nil
 
     config.paths["log"] = ENV["LOG_PATH"] if ENV["LOG_PATH"]
+
+    # Slimmer is used by the govuk_publishing_components gem, however it inserts itself
+    # automatically as middleware in the host Rails application
+    # Disable Slimmer middleware for Static and enable for use in gem only
+    if defined?(GovukPublishingComponents) && defined?(Slimmer)
+      config.middleware.delete Slimmer::App
+      GovukPublishingComponents::Engine.config.middleware.use Slimmer::App
```

Comment:
Again, my knowledge of Railties is basically nil, but would a conceptually neater approach to this be configuring `Slimmer` with which middleware to target (if possible)?

Reply:
Yes, I agree it would be neater. I decided against this because:

* it required a sizeable amount of refactoring in slimmer
* slimmer hasn't really been worked on fundamentally since 2013
* we aren't sure about the usefulness or shape of slimmer
* the refactor would only be necessary for this one use

Reply:
Sure fine by me.